pkg/kube/inject/webhook.go

```diff
@@ -97,9 +97,10 @@ const (
 )
 
 type WebhookConfig struct {
-	Templates  Templates
-	Values     ValuesConfig
-	MeshConfig *meshconfig.MeshConfig
+	RawTemplates RawTemplates
```

Comment:
Why this change? Doesn't appear to be used anywhere unless I missed it in the PR

Reply:
this might be from a prior iteration, let me double check. good call